initializr-service/src/main/resources/application.yml

```diff
@@ -278,8 +278,11 @@ initializr:
           description: Persistence support using MyBatis
           groupId: org.mybatis.spring.boot
           artifactId: mybatis-spring-boot-starter
-          version: 1.1.1
-          versionRange: 1.3.0.RELEASE
+          mappings:
+            - versionRange: 1.4.0.M2
+              version: 1.2.0
+            - versionRange: "[1.3.0.RELEASE,1.4.0.M2)"
```

Comment:
That's a bit unusual. Can we please do `1.3.x` for 1.1.1 and `1.4` for 1.2

Does `1.2` breaks with `1.4.0.M1` or something? Why did you pick that specific version?